client/commonFramework/update-preview.js

```diff
@@ -33,7 +33,7 @@ window.common = (function(global) {
 
 <link
   rel='stylesheet'
-  href='//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css'
+  href='//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css'
```

Comment:
This change has been made in `commonFramework` which i thought was now defunct? @BerkeleyTrue 

We do not use CDN's for font awesome, it will need to be updated through bower